### ISSUE TITLE
Get exception while parsing arrays with comments in the end

### ISSUE THREAD

**Issue by zensolution** [bertramdev/hcl4j#4]
I am receiving error while parsing the following hcl files:
````
cidr-blocks = [
  "10.12.22.22/24", # Test
  "10.65.0.0/23",  # Example
]
````

Parsing is good when I take "# Example" out. The error messages is like below,

com.bertramlabs.plugins.hcl4j.HCLParserException: Illegal character <(]) - state: 14> found on line: 4 col: 1
	at com.bertramlabs.plugins.hcl4j.HCLLexer.yylex(HCLLexer.java:993)

Thanks, in advance.

**Comment by davydotcom** [comment]
Thanks for reporting! Will definitely take a look

**Comment by davydotcom** [comment]
Resolved in 0.2.3

**Comment by davydotcom** [comment]
Resolved in 0.2.3
